cnd/src/cli.rs

```diff
@@ -3,7 +3,11 @@ use std::path::PathBuf;
 #[derive(structopt::StructOpt, Debug)]
 #[structopt(name = "COMIT network daemon")]
 pub struct Options {
-    /// Path to configuration file
+    /// Path to configuration file.
     #[structopt(short = "c", long = "config", parse(from_os_str))]
     pub config_file: Option<PathBuf>,
+
+    /// Path to secret seed file.
+    #[structopt(long = "secret-seed-file", parse(from_os_str))]
```

Comment:
Have a short option here? Maybe `s`?

Reply:
Implemented as suggested.

Reply:
I had another go at the typescript stuff but have not managed to get it working.  If anyone wants to pick this up in order to get it in before the end of sprint they are welcome.  Otherwise I'll try and typescript on Monday.

Reply:
@luckysori @D4nte @thomaseizinger - shameless bump to bring attention to why this is still WIP.

Reply:
I'll take a look :)

Reply:
Implemented serializing a pem file for you in the `cnd_runner` file. I didn't run all the tests but the first couple ones definitely pass :)

Reply:
Let's work together Monday?